src/api/test/unit/user_test.rb

```diff
@@ -98,6 +98,24 @@ def test_user_requests
     assert_equal 3, users(:fred).nr_of_requests_that_need_work
   end
 
+  def test_update_globalroles
+    user = User.find_by(login: "tom")
+    local_role = Role.create(title: "local_role", global: false)
```

Comment:
Sorry that I merged already. I still had a tab with this PR open and only saw your comment after the merge:(

You are right. The local_role was a leftover. I'll fix it in another commit.